pkgs/applications/emulators/retroarch/update_cores.py

```diff
@@ -50,6 +50,7 @@
     "dolphin": {"repo": "dolphin"},
     "dosbox": {"repo": "dosbox-libretro"},
     "dosbox-pure": {"repo": "dosbox-pure", "owner": "schellingb"},
+    "easyrpg": {"repo": "Player", "owner": "EasyRPG", "fetch_submodules": True, "rev": "0.8"},
```

Comment:
Why are you setting `rev` here? In general we prefer building from the current main branch, unless there is a strong reason not do to so (e.g.: the main branch is frequently broken).

Reply:
From what I understand, the standalone `easyrpg-player` in `nixpkgs` is pinned to `0.8` because of an incompatibility with the `fmt` lib. Because of this, one of its dependencies (`liblcf`) has not been updated either, so when building with the latest commit this error occurs:

```
error: builder for '/nix/store/ys55r8krbnnncwwsg2gvzsshwdyir4wz-libretro-easyrpg-unstable-2024-01-28.drv' failed with exit code 2;
       last 10 log lines:
       >    52 |         return HasRpg2k3BattleSystem() && lcf::Data::battlecommands.easyrpg_fixed_actor_facing_direction > 0;
       >       |                                                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > /build/source/src/feature.cpp: In function 'bool Feature::HasFixedEnemyFacingDirection()':
       > /build/source/src/feature.cpp:56:69: error: 'class lcf::rpg::BattleCommands' has no member named 'easyrpg_fixed_enemy_facing_direction'
       >    56 |         return HasRpg2k3BattleSystem() && lcf::Data::battlecommands.easyrpg_fixed_enemy_facing_direction > 0;
       >       |                                                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > make[2]: *** [CMakeFiles/EasyRPG_Player.dir/build.make:594: CMakeFiles/EasyRPG_Player.dir/src/feature.cpp.o] Error 1
       > make[2]: *** Waiting for unfinished jobs....
       > make[1]: *** [CMakeFiles/Makefile2:105: CMakeFiles/EasyRPG_Player.dir/all] Error 2
       > make: *** [Makefile:136: all] Error 2
```

`liblcf` seems to be used only by the standalone `easyrpg-player` so far. So, in order to maintain parity with the `nixpkgs` version, I replicated the version pinning and patchset here:

https://github.com/NixOS/nixpkgs/blob/5199001f7c631353399098ffd59a07bcaa77f3b9/pkgs/games/easyrpg-player/default.nix#L42-L59

I think patching `easyrpg-player` and `liblcf` could be a separate change.

Reply:
Can you add this information as a comment so it is easier to update it later?

Reply:
Added now.